### PR TITLE
sets up jest in backend

### PR DESCRIPTION
**Pull request by FelixPriori**
## Dependencies added in back-end
* Sequelize
* Jest
* Supertest

## Changes to package.json
We can now use `$ npm run test:watch` to start the tests.

## New dir & files
Adds `back-end/__tests__/endpoints.test.js` with basic test

### CODE REVIEW

File: back-end/__tests__/endpoints.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
+  it('Testing that endpoints are posting data', () => {
+    expect(1).toBe(1);
+  });
+});
```
Comment (cawel) [inline back-end/__tests__/endpoints.test.js:5]: Usually mocks are in such `__tests___` folder. A regular application test should probably be located in `back-end/tests`

File: back-end/__tests__/endpoints.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
+  it('Testing that endpoints are posting data', () => {
```
Comment (cawel) [inline back-end/__tests__/endpoints.test.js:2]: Ideally, tests should read as if it were in English.
* "it testing that endpoints are posting data" => not valid English
* "it tests that endpoints are posting data" => valid English

Comment (cawel) [inline back-end/__tests__/endpoints.test.js:2]: Ideally, test descriptions should read as if it were in English:

* "it testing that endpoints are posting data" => not valid English
* "it tests that endpoints are posting data" => valid English

File: back-end/__tests__/endpoints.test.js (hunk 1 above)
Comment (cawel) [inline back-end/__tests__/endpoints.test.js:5]: Usually mocks are in such `__tests___` folder. A regular application test should probably be located in `back-end/tests`

File: back-end/db/helpers.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
+/* eslint-disable camelcase */
```
Comment (cawel) [inline back-end/db/helpers.js:1]: Couldn't you remove that? 😢

File: back-end/__tests__/endpoints.test.js (hunk 1 above)
Comment (FelixPriori) [reply back-end/__tests__/endpoints.test.js:5]: noted. I'll adjust at next PR

Comment (cawel) [reply back-end/__tests__/endpoints.test.js:5]: Actually, it seems like some projects out there are using that `__tests__` folder. It's just that I haven't seen that often at all :/

**Pull request merged**